### DISCUSSION TITLE
Show Euro symbol after value

### DISCUSSION THREAD

**Discussion by CWempe** [grafana/grafana#45643] [Dashboards, Panels & Visualizations]
I would be great if the Eurosymbol could be placed after the value instead of before.

![image](https://user-images.githubusercontent.com/1270359/154843510-10d12d89-301a-4d67-b492-1082edcc4081.png)

In many countries like Germany this is the defined way.

wrong: € 123,45  
correct:  123,45 €

see: https://publications.europa.eu/code/en/en-370303.htm#:~:text=Dutch%2C%20Irish%20and%20Maltese.-,In%20all%20other%20official%20EU%20languages%20the%20order%20is%20reversed,-%3B%20the%20amount%20is%20followed

I know the work around is to define a custom unit.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82969. Feel free to continue the discussion around this there. Thank you!
